### ISSUE TITLE
Test improvement

### ISSUE THREAD

**Issue by DonCallisto** [APY/APYDataGridBundle#906]
Hello everybody,
in order to fulfill point 5 of https://github.com/APY/APYDataGridBundle/pull/877#issuecomment-267813654 I've started a [brand new branch ](https://github.com/APY/APYDataGridBundle/tree/test-improvement) (from [`symfony3` one](https://github.com/APY/APYDataGridBundle/tree/symfony3)) that will contain all commits for missing tests.

You are welcome if you want to collaborate as I'm not a master of PHPUnit and as I'm pretty new to this project.

Moreover it will be useful a list of non tested classes in order to check them while we do this task.

**Comment by DonCallisto** [comment]
@hmert maybe is better a PR with a WIP status in order to keep track of current files and commits? What do you think?

**Comment by hmert** [comment]
@DonCallisto Test improvement looks good for me. And do you merge that new test branch into symfony3 branch?

**Comment by DonCallisto** [comment]
@hmert yes, the plain is this as from that moment we could also do some refactor.
However I suppose I'm gonna need some help for "core" files; now I'm on more "peripheral" ones
